Dangerfile

```diff
@@ -19,3 +19,5 @@ no_changelog_entry = !git.modified_files.include?("CHANGELOG.md")
 if has_app_changes && no_changelog_entry && not_declared_trivial
   fail("Any changes to library code need a summary in the Changelog.")
 end
+
+jazzy.check fail: :modified
```

Comment:
@ilyapuchka shouldn't we check also for `:all` - added and modified?

> Except that it supposed to post warning only.
It is failing, because you pass `fail:`, we can pass `warn:` if we wanna show the warning only. But as I mentioned already I vote for fail 😄